### DISCUSSION TITLE
FinPolyLine vs FinLine (final boss)

### DISCUSSION THREAD

**Discussion by seyeint** [highfestiva/finplot#450] [Q&A]
Hey, quick question (probably last since this is one of the final features I need to close the project):

FinLine has the attribute points and with those I can work the line, whether updating or locating... but I can't do the same for FinPolyLine, aka a line that is created exactly the same but has interactive=True. 
I have 2 ways of solving this problem:

1) I learn how to access the points (coordinates) on a finpolyline and go from there;
2) If not possible due to the nature of the object itself being different, I would need to at least be able to access the text it has whenever I create or drag the line that can be seen below - usually tells the % from the origin. 

Either one of these work, maybe 2 is even easier for me but feel free to say how to work with both.

**Comment by highfestiva** [comment]
There is one function `finplot._draw_line_segment_text(polyline, segment, pos0, pos1)`, which you can overwrite (set) to a function you define yourself. The default implementation says something about time and percentage movement. This should take you all the way. If you simply want to get rid of the text, you just return an empty string.
